src/leo_object_storage_server.erl

```diff
@@ -430,9 +430,12 @@ handle_call({put = Method, #?OBJECT{addr_id = AddrId,
     Key_1 = ?gen_backend_key(StorageInfo#backend_info.avs_ver_cur,
                              AddrId, Key),
     {Reply, State_1} = put_1(Key_1, Object, State),
-
-    erlang:garbage_collect(self()),
-    reply(Method, Key, Reply, InTime, State_1);
+    case State#obj_server_state.is_gc_wip of
+        false ->
+            erlang:garbage_collect(self(), [{async, void}]);
```

Comment:
I've recognized we cannot use `erlang:garbage_collect/2` due to https://github.com/leo-project/leo_object_storage/pull/17#issuecomment-348422811

Reply:
I guess multiple invocations of erlang:garbage_collect/2 might hit the VM crash problem so this PR can avoid the problem by making it happen on only one process.